### PR TITLE
feat: [PLG-1569]: build credit flow

### PR DESCRIPTION
**Pull request by dhruvUpadhyay07**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by dhruvUpadhyay07** [comment]
retrigger cypress-cd

**Comment by dhruvUpadhyay07** [comment]
retrigger cypress-cv

**Comment by dhruvUpadhyay07** [comment]
retrigger cypress-pipeline

**Review by devinder-harness — APPROVED** [review]

File: src/modules/33-auth-settings/pages/subscriptions/overview/BuildCreditInfoTable.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        Header: NameHeader('common.expiryDateCredit'),
+        accessor: 'identifier',
+        disableSortBy: true,
+        width: '38%',
+        Cell: expiryTimeCell
+      },
+      {
+        Header: NameHeader('common.entitlement'),
+        accessor: 'serviceInstances',
+        width: '33%',
```
Comment (mayankVermaHarness) [inline src/modules/33-auth-settings/pages/subscriptions/overview/BuildCreditInfoTable.tsx:102]: The sum of widths of all columns is greater than 100.

File: src/modules/33-auth-settings/pages/subscriptions/overview/BuildCreditInfoTable.tsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+        className: pageCss.overUseClass
+      }
+    ]
+  }, [])
+  return (
+    <Card className={pageCss.outterCard}>
+      <Layout.Vertical spacing="xxlarge" flex={{ alignItems: 'stretch' }}>
+        <Layout.Horizontal spacing="small" flex={{ justifyContent: 'space-between' }} width={'100%'}>
+          <Layout.Vertical>
```
Comment (mayankVermaHarness) [inline src/modules/33-auth-settings/pages/subscriptions/overview/BuildCreditInfoTable.tsx:132]: `Layout.Vertical` not needed if it has only 1 child

File: src/modules/33-auth-settings/pages/subscriptions/overview/BuildCreditInfoTable.tsx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+        count: !isUndefined(data[0]?.expiryTime) ? moment(data[0]?.expiryTime).format('MMM DD YYYY') : '',
+
+        className: pageCss.overUseClass
+      }
+    ]
+  }, [])
+  return (
+    <Card className={pageCss.outterCard}>
+      <Layout.Vertical spacing="xxlarge" flex={{ alignItems: 'stretch' }}>
+        <Layout.Horizontal spacing="small" flex={{ justifyContent: 'space-between' }} width={'100%'}>
```
Comment (mayankVermaHarness) [inline src/modules/33-auth-settings/pages/subscriptions/overview/BuildCreditInfoTable.tsx:131]: `Layout.Horizontal` not needed if it has only 1 child

**Review by mayankVermaHarness — APPROVED** [review]

**Pull request merged**